packages/httpsnippet-client-api/test/index.test.ts

```diff
@@ -137,72 +128,6 @@ describe('httpsnippet-client-api', () => {
 
         expect(`${code}\n`).toStrictEqual(expected);
       });
-
-      it('should generate a functional snippet', async () => {
```

Comment:
These tests used a dist of the dynamic `api` entrypoint. I don't really have any interest in trying to get this extremely gross (my fault) `vm` sandboxing work working on that so I'm deleting all this. In the future it would probably be worth fleshing out a full integration suite for all these test cases but that's all really pertinent to `api` and not code snippet generation.

Reply:
would be cool to have some sort of mocking framework/library for easily testing these plugins

Reply:
yeah that'd be nice